lib/common/options.c

```diff
@@ -584,20 +585,40 @@ pcmk__format_option_metadata(const char *name, const char *desc_short,
             continue;
         }
 
+        g_string_append_printf(s, "    <parameter name=\"%s\">\n",
+                                  option_list[lpc].name);
+
         escaped_long = crm_xml_escape(option_list[lpc].description_long?
-                                         _(option_list[lpc].description_long) :
-                                          _(option_list[lpc].description_short));
-        escaped_short = crm_xml_escape(_(option_list[lpc].description_short));
+                                         option_list[lpc].description_long :
+                                          option_list[lpc].description_short);
+        escaped_short = crm_xml_escape(option_list[lpc].description_short);
 
-        g_string_append_printf(s, "    <parameter name=\"%s\">\n"
+        g_string_append_printf(s,
                                   "      <longdesc lang=\"en\">%s%s%s</longdesc>\n"
                                   "      <shortdesc lang=\"en\">%s</shortdesc>\n",
-                                  option_list[lpc].name,
                                   escaped_long,
                                   (option_list[lpc].values? "  Allowed values: " : ""),
                                   (option_list[lpc].values? option_list[lpc].values : ""),
                                   escaped_short);
-        free(escaped_long);
+
+#ifdef ENABLE_NLS
+        escaped_long = crm_xml_escape(option_list[lpc].description_long?
+                                         _(option_list[lpc].description_long) :
+                                          _(option_list[lpc].description_short));
+        escaped_short = crm_xml_escape(_(option_list[lpc].description_short));
+
+	locale=strtok(setlocale(LC_ALL,NULL),"_");
+	g_string_append_printf(s,
+                                  "      <longdesc lang=\"%s\">%s%s%s</longdesc>\n"
+                                  "      <shortdesc lang=\"%s\">%s</shortdesc>\n",
+                                  locale,
+				  escaped_long,
+                                  (option_list[lpc].values? "  Allowed values: " : ""),
+                                  (option_list[lpc].values? option_list[lpc].values : ""),
+                                  locale,
+				  escaped_short);
+#endif
+	free(escaped_long);
```

Comment:
The second free of escaped_long/escaped_short (or the free of new variables) would be inside the #endif